### DISCUSSION TITLE
How to use translateZ?

### DISCUSSION THREAD

**Discussion by gremo** [tailwindlabs/tailwindcss#6566] [Help]
I can't find any `translateZ` utility here: https://tailwindcss.com/docs/translate am I'm missing something?

It kinda "works" using `[transform:translateZ(50px)]`, but it will override ` -translate-x-1/2 -translate-y-1/2`

```html
<div class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 shadow-lg [transform:translateZ(50px)]"></div>
```

**Comment by sambauers** [comment]
I made this recently. It’s early days, but I think it has potential to fill this gap and others around 3D transforms.

https://github.com/sambauers/tailwindcss-3d

A bit lacking in documentation, but should make sense to most folks familiar with Tailwind transforms.
